src/js/models/menubar.js

```diff
@@ -219,7 +219,7 @@ define(function (require, exports, module) {
                 (document !== null) &&
                 !document.unsupported &&
                 (document.layers !== null) &&
-                (document.layers.selectedNormalized.size > 1),
+                (document.layers.selected.size > 1),
```

Comment:
`layers-selected-2+` was only used by align commands. Normalizing gets rid of selected children of other selected layers, which is not what we want for align.

Since Align is the only client of `layers-selected-2+`, I went ahead and changed only that, but I think your confusion is justification enough that we may need a separate rule for align and leave all `layer-selected-#` rules with `selectedNormalized`

Will revisit.